ecosystem/typescript/sdk/src/aptos_types/transaction.ts

```diff
@@ -208,6 +208,39 @@ export class EntryFunction {
   }
 }
 
+export class MultiSig {
+  /**
+   * Contains the payload to run a multisig account transaction.
+   * @param multisig_address The multisig account address the transaction will be executed as.
+   * @param transaction_payload The payload of the multisig transaction. This is optional when executing a multisig transaction
+   *  whose payload is already stored on chain.
+   */
+  constructor(
+    public readonly multisig_address: AccountAddress,
+    public readonly transaction_payload?: EntryFunction | undefined,
+  ) {}
+
+  serialize(serializer: Serializer): void {
+    this.multisig_address.serialize(serializer);
+    if (this.transaction_payload === undefined) {
+      serializer.serializeBool(false);
+    } else {
+      serializer.serializeBool(true);
+      this.transaction_payload.serialize(serializer);
+    }
```

Comment:
Can we add some comments that this is serializing an Option? Otherwise, the `serializeBool` is confusing here

Reply:
Updated